resolve-environment/action.yml

```
@@ -3,6 +3,7 @@ description: '[Experimental] Attempt to infer a build environment suitable for a
 author: 'GitHub'
 inputs:
   token:
+    description: "Token for accessing /code-scanning/analysis endpoints (used for ???)"
```

Same as other comments!
```suggestion
    description: "GitHub token to use for authenticating with this instance of GitHub. The token needs the `security-events: write` permission."
    required: false
```